app/helpers/content_items_helper.rb

```diff
@@ -0,0 +1,9 @@
+module ContentItemsHelper
+  def page_title
```

Comment:
As this helper is shared across all controllers I would rename it to something more specific, or I would configure Rails to not load all helpers in all views and be explicit at controller level.